app/retail/templates/shared/head.html

```diff
@@ -32,6 +32,7 @@
     <link rel="stylesheet" href="{% static "v2/css/timeline.css" %}" />
     <link rel="stylesheet" href="{% static "v2/css/carousel.css" %}" />
     <link rel="stylesheet" href="{% static "v2/css/faucet.css" %}" />
+    <link rel="stylesheet" href="{% static "v2/css/ideas.css" %}" />
```

Comment:
Thoughts on moving this to just the `ideas_list.html` page cause it won't be used anywhere else ? Would make the requests on the other pages a tad bit lighter

Reply:
I think ideas_detail may also eventually use stuff from v2/css/ideas.css 🤔

Reply:
If ideas_detail.html doesn't use stuff from ideas.css I will yank this link out and drop in the ideas_list.html per your guidance 👍🏻